.travis.yml

```diff
@@ -22,6 +22,10 @@ env:
     - CI_NODE_INDEX=3
     - CI_NODE_INDEX=4 KARMA="true" GITHUB_DEPLOY="true"
 
+before_install:
+  - gem install bundler
```

Comment:
Agree on being deterministic. I wanted to address that with CodeClimate but your solution works perfectly.

Ideally we should have the same bundler version everywhere but I think that while developing what can become a blocker is having too diverse versions in CI and your dev env, potentially leading to different execution results.

Regarding backward compatibility I don't know if it's something we need to care about for bundler. You just need to ensure that it'll work in production.